src/main/scala/mesosphere/marathon/core/health/impl/HealthCheckActor.scala

```diff
@@ -84,12 +83,11 @@ private[health] class HealthCheckActor(
     }
 
   def dispatchJobs(): Unit = {
-    import TemporarilyUnreachable.isUnreachable
-
     log.debug("Dispatching health check jobs to workers")
     taskTracker.appTasksSync(app.id).foreach { task =>
+
       task.launched.foreach { launched =>
-        if (launched.runSpecVersion == app.version && launched.hasStartedRunning && !isUnreachable(task)) {
+        if (launched.runSpecVersion == app.version && launched.hasStartedRunning && !task.isUnreachable) {
```

Comment:
This can simply be
`if (launched.runSpecVersion == app.version && task.isRunning)` 😁